src/components/elements/headers.tsx

```diff
@@ -5,14 +5,19 @@ type Props = HtmlHTMLAttributes<HTMLHeadingElement>
 
 const headingLinkClasses = "[&_a]:hocus:underline"
 
+const handleHeadingClasses = (classes: string | undefined, headingClasses: string): string[] => {
+  if (!classes) {
+    return [headingClasses]
+  }
+
+  return classes.match(/(type-\d|text-\d)/g) ? [classes] : [headingClasses, classes]
```

Comment:
Didn't you want to use the regex I provided?

Reply:
I do! For some reason that slipped my mind during the rework 🤦

PR should be all up to date now!